app/workers/http_multi.rb

```diff
@@ -4,7 +4,7 @@
 
 module Workers
   class HttpMulti < Base
-    sidekiq_options queue: :http
+    sidekiq_options queue: :high
```

Comment:
These jobs are extremely long running, they shouldn't block other jobs, so this should be `medium` or even `low`.